<ui version="4.0" >
 <class>qdCopyMove</class>
 <widget class="QDialog" name="qdCopyMove" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlSource" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Searching for files,</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlDestination" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>please wait.</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="qpbTotal" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>0</number>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
   <property name="textVisible" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>270</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Background</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
